app/App/Footer/RequestCommand/index.js

```diff
@@ -55,6 +55,17 @@ class RequestCommand extends React.Component {
     if (req.status === 'confirmed') requestClass += ' signerRequestConfirmed'
     else if (error) requestClass += ' signerRequestError'
 
+
+    const cancelableRequest = ![
```

Comment:
can we move this out to a shared location (perhaps `/resources/domain`)? I want to make sure we centralize all this logic so that we don't miss anything when we start to change the way we represent requests

Reply:
👍 moved this one